install_config/web_console_customization.adoc

```diff
@@ -334,6 +334,38 @@ endif::[]
 
 endif::[]
 
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
+
+Sometimes features are available in tech preview. By default, these features are
```

Comment:
s/tech preview/Technology Preview/

Also link the words to https://access.redhat.com/support/offerings/techpreview.